docs/studio/groups/group-rules.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: "Group Rules"
+description: "Configure the resources accessible to your groups."
+icon: "database"
+---
+
+A **group rule** defines the roles and associated resources that determine what group members and API keys can access, as well as the level of permissions granted.
+
+<Frame>
+    <img src="/images/studio/group-rules.png" />
+</Frame>
+
+## Roles
+
+<Note>
+    If a role is assigned to a single resource and that resource is deleted, the role will default to granting access to all resources in the organization.
```

Comment:
Maybe rephrase and explain in the beginning, that a rule without explicit permissions always has access to all resources within the organization.